HW_1/Task_2.py

```diff
@@ -0,0 +1,16 @@
+
+#  Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+
+n = int(input("Введите число: "))
+sum = 0
+
+while n > 0:
+    a = n % 10
+    sum = sum + a
```

Comment:
Это можно соединить и записать через сокращение
```suggestion
    sum += n % 10
```